github/teams_discussion_comments.go

```diff
@@ -39,12 +39,12 @@ type DiscussionCommentListOptions struct {
 	Direction string `url:"direction,omitempty"`
 }
 
-// ListComments lists all comments on a team discussion.
+// ListCommentsByID lists all comments on a team discussion.
```

Comment:
For each of these method comments, let's please follow the format in the other updates and add:

```
by team ID.
```

or 

```
by team slug.
```

to the end of the descriptions.